airgun/views/container.py

```diff
@@ -0,0 +1,150 @@
+from widgetastic.widget import (
+    Text,
+    TextInput,
+    View,
+    Checkbox,
+)
+from airgun.views.common import (
+    BaseLoggedInView,
+    SearchableViewMixin,
+    SatTab,
+)
+from airgun.widgets import (
+    FilteredDropdown,
+    ActionsDropdown,
+    SatTable,
+    MultiSelect,
+    EditableEntryCheckbox,
+)
+from widgetastic_patternfly import BreadCrumb
+
+
+class ContainerView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Containers']")
+    new = Text("//a[contains(@href, '/containers/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Status': Text("./span[text()='On']"),
+            'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
+        }
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class ContainerCreateView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+            breadcrumb_loaded
+            and self.breadcrumb.locations[0] == 'Steps'
+            and self.breadcrumb.read() == 'Create container'
+        )
+
+    @View.nested
+    class ContainerPreliminaryCreateView(BaseLoggedInView):
```

Comment:
since these classes are somewhat like tabs inside ContainerCreateView and you have to pass their names for creation, pls make them shorter to avoid such long lines in tests:
```python
             'ContainerPreliminaryCreateView.compute_resource.deploy_on': deploy_on,
             'ContainerImageCreateView.content_view.lifecycle_environment': lifecycle_environment,
             'ContainerImageCreateView.content_view.content_view': content_view,
             'ContainerImageCreateView.content_view.repository': repo.name,
             'ContainerImageCreateView.content_view.tag': '7',
             'ContainerImageCreateView.content_view.capsule': settings.server.hostname,
```
i'd rather call them `preliminary`, `image`, `configuration` and `environment` (lowercase btw).